examples/alphaTestExamples/inventoryNamespace.md

```diff
@@ -99,7 +99,7 @@ expectedOutputLine "1"
 invName=$(kubectl get cm -n test-namespace --selector='cli-utils.sigs.k8s.io/inventory-id' --no-headers | awk '{print $1}')
 
 # There should be one config map that is not the inventory object
-kubectl get cm -n test-namespace --selector='!cli-utils.sigs.k8s.io/inventory-id' --no-headers | wc -l > $OUTPUT/status
+kubectl get cm -n test-namespace --selector='name=test-config-map-label' --no-headers | wc -l > $OUTPUT/status
```

Comment:
There is a [new `ConfigMap` `kube-root-ca.crt`](https://v1-20.docs.kubernetes.io/docs/setup/release/notes/#introducing-rootcaconfigmap) now, so made this more robust.
```
NAME               DATA   AGE
cm-a               0      2s
cm-b               0      2s
cm-c               0      2s
kube-root-ca.crt   1      7s
```

Reply:
Previously `kind` was creating Kubernetes v1.19.1 and now it creates v1.21.1.